bundler/spec/commands/newgem_spec.rb

```diff
@@ -28,6 +28,7 @@ def bundle_exec_rubocop
   let(:require_path) { "mygem" }
 
   before do
+    global_config "BUNDLE_GEM__CHANGELOG" => "false"
```

Comment:
Could you confirm whether this line is actually necessary? Since it's not needed for other analogous flags, I have the feelling that it's not needed here either.

Reply:
You are correct! Removed it 👍